src/_includes/partials/footer.html

```diff
@@ -2,36 +2,36 @@
+                        href='mailto:web3community@gmail.com' rel='noreferrer' target='_blank' title='mail us'></a>
 
-                <a aria-hidden='true'
-                    class='text-white fa fa-instagram fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
-                    href='https://www.instagram.com/web3community/' rel='noreferrer' target='_blank'
-                    title='follow us'></a>
+                    <a aria-hidden='true'
+                        class='text-white fa fa-instagram fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        href='https://www.instagram.com/web3community/' rel='noreferrer' target='_blank'
+                        title='follow us'></a>
 
-                <a aria-hidden='true'
-                    class='text-white fa fa-twitter fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
-                    href='https://twitter.com/web3community' rel='noreferrer' target='_blank' title='follow us'></a>
+                    <a aria-hidden='true'
+                        class='text-white fa fa-twitter fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        href='https://twitter.com/web3community' rel='noreferrer' target='_blank' title='follow us'></a>
 
-                <a aria-hidden='true'
-                    class='text-white fa fa-linkedin fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
-                    href='https://www.linkedin.com/company/web3community' rel='noreferrer' target='_blank'
-                    title='follow us'></a>
+                    <a aria-hidden='true'
+                        class='text-white fa fa-linkedin fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        href='https://www.linkedin.com/company/web3community' rel='noreferrer' target='_blank'
+                        title='follow us'></a>
 
-                <a aria-hidden='true'
-                    class='text-white fa fa-github fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
-                    href='https://github.com/web3community' rel='noreferrer' target='_blank' title='follow us'></a>
-            </div>
-            <!--socials icons ended-->
+                    <a aria-hidden='true'
+                        class='text-white fa fa-github fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        href='https://github.com/web3community' rel='noreferrer' target='_blank' title='follow us'></a>
+                </div>
+                <!--socials icons ended-->
 
-            <!--menu start-->
+                <!--menu start-->
```

Comment:
I think the indentation was fine. Please change it back as to how it was (same in other lines above).

Reply:
yeah, I think I have Prettier configed to run on save for all projects on my workspace... sorry about that.